keyboards/pabile/p42/p42.h

```diff
@@ -0,0 +1,33 @@
+/*
+Copyright 2020 Pabile
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+#include "quantum.h"
+
+#define LAYOUT_default( \
```

Comment:
This should be named `LAYOUT` as it's the only possible option.

```suggestion
#define LAYOUT( \
```

Reply:
thanks for the clarification, noroadsleft. i thought i should change to the address the error i am receiving. reverting to LAYOUT()